integration-test/src/test/java/com/datastrato/gravitino/integration/test/spark/iceberg/SparkIcebergCatalogIT.java

```diff
@@ -31,4 +33,15 @@ protected boolean supportsSparkSQLClusteredBy() {
   protected boolean supportsPartition() {
     return false;
   }
+
+  @Test
+  void testIcebergDeleteOperation() {
```

Comment:
could you add test to insert multiple datas to the table and  only delete partial of the data?

Reply:
sure

Reply:
and I prefer the delete test to `SparkCommonIT`, because JDBC table could reuse it.

Reply:
updated